relayer-engine/src/executor/executorHarness.ts

```diff
@@ -32,8 +32,9 @@ import {
 // todo: add to config
 const DEFAULT_WORKER_RESTART_MS = 10 * 1000;
 const DEFAULT_WORKER_INTERVAL_MS = 500;
-const MAX_ACTIVE_WORKFLOWS = 10;
-const SPAWN_WORKFLOW_INTERNAL = 500;
+export const MAX_ACTIVE_WORKFLOWS = 10;
```

Comment:
I think this is a merge artifact

Reply:
I'm using in this in the executor to see if we should process a job and in storage to check the delayed queue. I'm not happy about it, we probably need a refactor on the storage service.